docker-for-windows/edge-release-notes.md

```diff
@@ -11,6 +11,19 @@ This page contains information about Docker Desktop Edge releases. Edge releases
 For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.5.2
+2020-11-24
+
+> [Download](https://desktop.docker.com/win/edge/50073/Docker%20Desktop%20Installer.exe)
+
+### New
+
+- Fixed an unexpected EOF error when trying to start a non-existing container with `-v /var/run/docker.sock:`. See [docker/for-mac#5025](https://github.com/docker/for-mac/issues/5025).
```

Comment:
@lorenrh Looks like this entry is incorrectly added to the New section

Reply:
Yes, and the ones that were in New have disappeared.

Reply:
my bad! It's fixed now.